### PR TITLE
IPv6 support for govc

### PR DESCRIPTION
**Pull request by hreese**
I screwed up my branches; here is a new PR for IPv6 support for govc. See also #2151.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,8 @@ require (
 	github.com/kr/pretty v0.1.0
 	github.com/kr/text v0.1.0 // indirect
 	github.com/vmware/vmw-guestinfo v0.0.0-20170707015358-25eff159a728
+	golang.org/x/tools v0.0.0-20201007032633-0806396f153e // indirect
```
Comment (dougm) [inline go.mod:13]: Doesn't look like the go.mod or go.sum changes are needed, can you revert those?

**Review by dougm — COMMENTED** [review]
Thanks @hreese , lgtm, just need to revert the go.mod and go.sum changes.

File: go.mod (hunk 1 above)
Comment (hreese) [reply go.mod]: Done. Looks like my IDE added some comfort features :-)

**Review by dougm — APPROVED** [review]
Looks great, thanks @hreese !

**Pull request merged**